### DISCUSSION TITLE
Other resources, webs, libraries or info to be checked.

### DISCUSSION THREAD

**Discussion by dprada** [uibcdf/OpenPharmacophore#6] [General]
Let's use this space to add webs, libraries, or resources that have to be checked at some point.

**Comment by dprada** [comment]
I had this list of webs to be checked:

https://github.com/meddwl/psearch     
https://github.com/ci-lab-cz/pharmd     
https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/412     
https://www.vls3d.com/index.php/links/chemoinformatics/virtual-screening/ligand-based-virtual-screening    
https://click2drug.org/directory_LigandBasedScreening.php    
https://www.schrodinger.com/products/phase    
http://www.rdkit.org/     
https://www.3ds.com/products-services/biovia/products/molecular-modeling-simulation/biovia-discovery-studio/pharmacophore/     
https://www.vlifesciences.com/products/VLifeMDS/Molsign.php     
https://www.researchgate.net/post/Can-you-recommend-any-free-software-for-pharmacophore-modelling     
https://www.researchgate.net/post/What_is_the_best_free_software_for_Pharmacophore_mapping2     
https://github.com/pharmai/plip    
http://www.inteligand.com/pubs/050606_wolber_iccs_noordwijkerhout.pdf    
https://www.chemcomp.com/moe/help/2018/index_tut.html

**Reply by Daniel-Ibarrola** [reply]
I think this java library is worth checking, it contains classes for pharmacophores:

https://cdk.github.io/
